src/modules/map/components/LocationPopup.vue

```diff
@@ -215,23 +229,36 @@ async function updateQrCode(url) {
                 <a :href="$t('elevation_href')" target="_blank">{{ $t('elevation') }}</a>
             </LocationPopupCopySlot>
 
-            <div class="location-popup-link location-popup-coordinates-label">
-                {{ $t('share_link') }}
-            </div>
-            <div class="location-popup-link location-popup-coordinates-data">
-                <LocationPopupCopyInput
-                    :value="shareLinkUrlDisplay"
-                    data-cy="location-popup-link-bowl-crosshair"
-                />
-            </div>
         </div>
-        <div class="location-popup-qrcode">
-            <img
-                v-if="qrCodeImageSrc"
-                :src="qrCodeImageSrc"
-                alt="qrcode"
-                data-cy="location-popup-qr-code"
-            />
+        <div class="menu-share-embed">
+            <button
+                class="btn btn-light btn-sm embedded-button"
+                data-cy="menu-share-embed-button"
+                @click="toggleEmbedSharing"
+            >
+                <FontAwesomeIcon :icon="`caret-${showEmbedSharing ? 'down' : 'right'}`" />
+                <span class="ms-2">{{ $t('share_link') }}</span>
+            </button>
+            <CollapseTransition :duration="100">
+                <div v-show="showEmbedSharing" class="p-2 card border-light bg-light">
+                    <!-- eslint-disable vue/no-v-html-->
```

Comment:
Why disabling the eslint n-v-html here? Should not be needed and could be dangerous